lib/poxa/event_handler.ex

```diff
@@ -62,7 +62,8 @@ defmodule Poxa.EventHandler do
   The event data can't be greater than 10KB
```

Comment:
Since now this limit is configurable, could you also update this comment to reflect that this is just the default value?